TaskOne/hash.js

```diff
@@ -0,0 +1,25 @@
+var crypto = require("crypto");
+var shaAlgorithm = "sha1";
+var mdAlgorithm = "md5";
+const fs = require("fs");
+function generateHash(filename) {
+  s = fs.ReadStream(filename);
+  var shasum = crypto.createHash(shaAlgorithm);
+  var mdsum = crypto.createHash(mdAlgorithm);
+  s.on("data", function (data) {
+    shasum.update(data);
+    mdsum.update(data);
+  });
+  s.on("end", function () {
+    var shash = shasum.digest("hex");
+    var mhash = mdsum.digest("hex");
```

Comment:
always choose a good readable names..

you could have simply choosen md5 and sha1 as variables or something like md5Hash, sha1Hash.

Suggestion:
`Good Code is when others can understand as they read !!`
make it a habit to write clear and relevant names always. Its ok to write longStrings !!